src/panels/config/dashboard/ha-config-updates.ts

```diff
@@ -31,68 +20,83 @@ export const SUPERVISOR_UPDATE_NAMES = {
 class HaConfigUpdates extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
-  @state() private _supervisorUpdates?: SupervisorAvailableUpdates[];
+  @property({ type: Boolean }) public narrow!: boolean;
 
-  @state() private _error?: string;
+  @property({ attribute: false })
+  public supervisorUpdates?: SupervisorAvailableUpdates[] | null;
 
-  protected firstUpdated(changedProps: PropertyValues): void {
-    super.firstUpdated(changedProps);
-    this._loadSupervisorUpdates();
-  }
+  @state() private _showAll = false;
 
   protected render(): TemplateResult {
+    if (!this.supervisorUpdates) {
+      return html``;
+    }
+
+    const updates =
```

Comment:
- If there are 3 or fewer updates show all
- If there are 4 or more, show 2 updates and give a link to expand